packages/cli/src/utils/identity.ts

```diff
@@ -34,6 +35,7 @@ export abstract class ClaimCommand extends BaseCommand {
     try {
       cli.action.start(`Read Metadata from ${filePath}`)
       const data = IdentityMetadataWrapper.fromFile(this.kit, filePath)
+      // Perhaps should check here that `--from` and data.data.meta.address match
```

Comment:
That's no longer the case right? From can be a signer?

Reply:
Yeah, so check that it's the address or one of the signers.